test/unit/lib/classes/PluginManager.test.js

```diff
@@ -1837,27 +1837,47 @@ describe('PluginManager', () => {
         localPath: serviceDir,
         modules: ['local-plugin'],
       });
-      // Had to use contructor.name because the class will be loaded via
+      // Had to use constructor.name because the class will be loaded via
       // require and the reference will not match with SynchronousPluginMock
       expect(pluginManager.plugins).to.satisfy((plugins) =>
         plugins.some((plugin) => plugin.constructor.name === 'SynchronousPluginMock')
       );
     });
 
-    it('should load plugins from custom folder outside of serviceDir', () => {
-      serviceDir = path.join(tmpDir, 'serverless-plugins-custom');
-      const localPluginDir = path.join(serviceDir, 'local-plugin');
-      installPlugin(localPluginDir, SynchronousPluginMock);
+    describe('#getLocalPluginsPathPatterns()', () => {
+      it('should return empty list without any plugins', () => {
```

Comment:
All new tests should be configured with `runServerless`, so if that case is converted by added `runServerless` run, then simply let's remove this test. Otherwise it'll be great to ensure it's covered with `runServerless` based test